draft-ietf-cose-merkle-tree-proofs.md

```diff
@@ -655,6 +621,17 @@ Provisional assignments to expired drafts MUST be removed from the registry.
 - Points assigned in this registry MUST have references that match the COSE Verifiable Data Structures registry.
 It is not permissible to assign points in this registry, for which no Verifiable Data Structure entry exists.
 
+# Acknowledgements {#Acknowledgements}
+
+We would like to thank
+Maik Riechert,
+Jon Geater,
+Mike Jones,
```

Comment:
```suggestion
Michael B. Jones,
```

Too many people share my name, so I use this moniker professionally.  In person, of course, I'm just "Mike".